<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RH03703" failures="1" tests="5" name="Default test(failed)" time="0.151" errors="0" timestamp="2023-04-04T12:29:43 GST">
  <testcase classname="vtiger.Practice.RetryAnalyserPractice" name="sample" time="0.046">
    <skipped/>
  </testcase> <!-- sample -->
  <testcase classname="vtiger.Practice.RetryAnalyserPractice" name="sample" time="0.005">
    <skipped/>
  </testcase> <!-- sample -->
  <testcase classname="vtiger.Practice.RetryAnalyserPractice" name="sample" time="0.004">
    <skipped/>
  </testcase> <!-- sample -->
  <testcase classname="vtiger.Practice.RetryAnalyserPractice" name="sample" time="0.004">
    <skipped/>
  </testcase> <!-- sample -->
  <testcase classname="vtiger.Practice.RetryAnalyserPractice" name="sample" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at vtiger.Practice.RetryAnalyserPractice.sample(RetryAnalyserPractice.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- sample -->
</testsuite> <!-- Default test(failed) -->
